ruma-client-api/Cargo.toml

```diff
@@ -37,3 +37,4 @@ matches = "0.1.8"
 [features]
 unstable-pre-spec = []
 unstable-synapse-quirks = []
+unstable-exhaustive-types = []
```

Comment:
Can these be sorted alphabetically? :)

Reply:
I originally prioritized keeping the diff short, but that can be easily done.

Reply:
The diff won't be any longer if this is inserted two lines above. This doesn't seem to be adressed?

Reply:
I thought this comment was on another file, whoops. It should be resolved in the latest push.